integreat_cms/firebase_api/firebase_statistic.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from integreat_cms.cms.models.abstract_base_model import AbstractBaseModel
+
+
+class FirebaseStatistic(AbstractBaseModel):
+    """
+    Data model representing a push notification translation
+    """
+
+    date = models.DateField()
+    region = models.CharField(max_length=100)
+    language = models.CharField(max_length=2)
```

Comment:
```suggestion
    language_slug = models.CharField(max_length=2)
```
I'd rather call this slug